src/backend/db/videos.js

```diff
@@ -118,6 +118,6 @@ export const videos = [
       "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
     creator: "Soham Shah",
     watchLater: false,
-    history: false,
+    liked: false,
```

Comment:
Not sure why you are using this attribute, you can add a video to the liked and always run a find to know if it is liked or not, basically whether if it's present in the liked array or not.